1209/5284/index.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: pid
+title: nRFMicro Bootloader
+owner: joric
+license: MIT
+site: https://github.com/adafruit/Adafruit_nRF52_Bootloader/pull/342
+source: https://github.com/joric/nrfmicro
+---

```

Comment:
Please add a description here. (I'm not sure why you deleted it.)

Reply:
36.6% of records here don't have a description (734 records total, 471 records with description and 272 records without description).